app/src/main/res/values/strings.xml

```diff
@@ -25,4 +26,7 @@
     <string name="TextShouldNotBeBlank">"Text should not be blank"</string>
     <string name="TextIsTooLong">"Text is too long"</string>
     <string name="TextIsValid">""</string>
+    <string name="PartyScreenFabAddIcon">Fab add icon</string>
```

Comment:
This values should indicates what this fab exactly doing :) Fab add icon is worthless in case of real accessibility :)